dev/start

```diff
@@ -34,6 +38,32 @@ EOF
   exit
 }
 
+function configure_oidc_providers() {
+  enable_oidc_authenticators
+
+  # Start conjur again, since it is recreating by docker-compose because of dependency with keycloak
+  start_conjur
```

Comment:
`restart_conjur`?

Reply:
Why restart? we are running the command : `conjur conjurctl server` so we are starting the server

Reply:
sorry, i thought you are restarting the container to get the keycloak vars or something